src/jit/lclvars.cpp

```diff
@@ -5985,11 +5985,13 @@ void Compiler::lvaAssignFrameOffsetsToPromotedStructs()
         //
         if (varDsc->lvIsStructField
 #ifndef UNIX_AMD64_ABI
+#if !defined(_TARGET_ARM_) || defined(LEGACY_BACKEND)
             // For System V platforms there is no outgoing args space.
```

Comment:
Can you explain why this doesn't apply to RyuJIT/ARM32? It would be worthwhile updating the surrounding comments with this explanation.